<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monolayer</class>
 <widget class="QMainWindow" name="Monolayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>547</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>790</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Monolayer_2">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Monolayer Ribbon</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="CalcBands">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Bandstructure</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalcEdgeL">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark lower edge</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>287</width>
      <height>326</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Number of unit cells</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="NUnit">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>t</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="Hoppingt">
       <property name="whatsThis">
        <string extracomment="INtra layer hopping"/>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>3.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Magnetic Field Type</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="SelectBMono"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="Magneticfield">
       <property name="toolTipDuration">
        <number>-2</number>
       </property>
       <property name="whatsThis">
        <string extracomment="Magnetic field strength"/>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="minimum">
        <double>-5.000000000000000</double>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="OnsiteV">
       <property name="whatsThis">
        <string extracomment="On-site potential"/>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Edge termination</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QComboBox" name="SelectEdge"/>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="Haldane">
       <property name="toolTipDuration">
        <number>-2</number>
       </property>
       <property name="whatsThis">
        <string extracomment="Magnetic field strength"/>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="minimum">
        <double>-5.000000000000000</double>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Haldane</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="AntiHaldane">
       <property name="toolTipDuration">
        <number>-2</number>
       </property>
       <property name="whatsThis">
        <string extracomment="Magnetic field strength"/>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedKingdom"/>
       </property>
       <property name="minimum">
        <double>-5.000000000000000</double>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Anti-Haldane</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="CalcEdgeU">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark upper edge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EdgeStates">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge states</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LDOS">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>440</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>LDOS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DOS">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>410</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>DOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of k points</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Energy window</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numberofKop">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Operators</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delta</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="delta">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>470</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-2</number>
    </property>
    <property name="whatsThis">
     <string extracomment="Magnetic field strength"/>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
    <property name="minimum">
     <double>-5.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.010000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="current">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Current</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="energy">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-2</number>
    </property>
    <property name="whatsThis">
     <string extracomment="Magnetic field strength"/>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedKingdom"/>
    </property>
    <property name="minimum">
     <double>-5.000000000000000</double>
    </property>
    <property name="maximum">
     <double>5.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMaster_Project">
    <property name="title">
     <string>Master Project</string>
    </property>
    <addaction name="actionMonolayer"/>
   </widget>
   <addaction name="menuMaster_Project"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMonolayer">
   <property name="text">
    <string>Monolayer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
